stable/openiban/templates/deployment.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "openiban.fullname" . }}
+  labels:
+    app: {{ template "openiban.name" . }}
```

Comment:
use updated labels best practices: https://github.com/helm/helm/blob/master/docs/chart_best_practices/labels.md (for all resources)

Reply:
Let's see what comes out of https://github.com/helm/charts/issues/7550

Reply:
No response on that - so let's keep it